environments/skeleton/{{cookiecutter.environment}}/terraform/compute/variables.tf

```diff
@@ -67,3 +67,8 @@ variable "root_volume_size" {
 variable "security_group_ids" {
     type = list
 }
+
+variable "k3s_token" {
+    description = "Random cryptographically secure string for K3s token (must be set by ../compute.tf)"
```

Comment:
I don't think we need descriptions in this file, where the vars are just passed through from the outer module